### DISCUSSION TITLE
[6일차-윤경]display의 종류

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#24] [프론트 면접 질문]
display의 종류 4가지를 말하시오?

**Comment by skyhanull** [comment]
display의 종류로 none, lnline,block,lnline-block 4가지가 있습니다.

**Comment by ez1n** [comment]
display 속성 : 요소를 어떻게 보여줄지 결정 - **block, inline, inline-bock, none**

1. block : 가로 영역을 모두 채움 (한 문단을 차지) - block 요소 다음에 등장하는 태그는 줄바꿈 된 것처럼 보인다.
⇒ `div` `p` `h` `li` 등이 해당됨
    
2. inline : 줄바꿈되지 않고 width와 height을 지정할 수 없음
⇒ `span` `b` `i` `a` 등이 해당됨
    
3. inline-block : block과 inline의 중간 형태 - 줄바꿈이 되지 않지만 크기 지정이 가능함
4. none : 요소를 렌더링하지 않음
 ⇒ `visibility: hidden` 은 영역을 차지하면서 보이지 않는것과 달리 영역도 차지하지 않음!
